test/e2e/index.test.ts

```diff
@@ -104,13 +115,15 @@ describe('e2e', () => {
       await renderFixture('Rating/BasicRating');
 
       await page.focus('input[name="rating-test"]:checked');
-      expect(await page.evaluate(() => document.activeElement?.getAttribute('value'))).to.equal(
-        '1',
-      );
+      await expect(page.evaluateHandle(() => document.activeElement)).toHaveAttribute('value', '1');
       await page.keyboard.press('ArrowLeft');
-      expect(await page.evaluate(() => document.activeElement?.getAttribute('value'))).to.equal('');
+      await expect(page.evaluateHandle(() => document.activeElement)).to.toHaveAttribute(
+        'value',
+        '',
+      );
       await page.keyboard.press('ArrowLeft');
-      expect(await page.evaluate(() => document.activeElement?.getAttribute('value'))).to.equal(
+      await expect(page.evaluateHandle(() => document.activeElement)).to.toHaveAttribute(
```

Comment:
`evaluateHandle` is too powerful for regular usage I think. But it mirrors the `expect(document.activeElement).to.have.attribute` quite nicely. The alternative would be a custom API to get the focused element and neither playwright nor testing-library has one (since its implementation is trivial) so I'm hesitant to add one.
Let's just hope people don't abuse `evaluateHandle` more than they abuse `document.querySelector`.